examples/echobot/echo.go

```diff
@@ -83,7 +83,8 @@ func echo(ctx context.Context, addr, pass string, xmlIn, xmlOut io.Writer, logge
 	}
 
 	return s.Serve(xmpp.HandlerFunc(func(t xmlstream.TokenReadEncoder, start *xml.StartElement) error {
-		d := xml.NewTokenDecoder(t)
+		d := xml.NewTokenDecoder(xmlstream.MultiReader(xmlstream.Token(*start), t))
+		d.Token()
```

Comment:
We still want to do error handling here. Errors may currently be impossible, but if we're wrong about that or somehow they become possible in the future we want them reported.